OMCompiler/README.Linux.md

```diff
@@ -3,14 +3,15 @@
 ## Debian/Ubuntu Compile Cheat Sheet (or read on for the full guide)
 
 ```bash
-echo deb http://build.openmodelica.org/apt bionic nightly | sudo tee -a /etc/apt/sources.list
-echo deb-src http://build.openmodelica.org/apt bionic nightly | sudo tee -a /etc/apt/sources.list
+echo Linux name: `lsb_release --short --codename`
+echo deb http://build.openmodelica.org/apt `lsb_release --short --codename` nightly | sudo tee -a /etc/apt/sources.list
+echo deb-src http://build.openmodelica.org/apt `lsb_release --short --codename` nightly | sudo tee -a /etc/apt/sources.list
```

Comment:
Not every Linux is supported I guess. We could add something like:
```bash
# See http://build.openmodelica.org/apt/dists/ for supported distributions.
```
So if you are using some strange Linux you could still try `bionic` an so on.

Reply:
Ah, i merged it before refreshing the page. I'll add some text about this.